ddpui/management/commands/saveflowrunlogstos3.py

```diff
@@ -70,40 +78,103 @@ def fetch_grouped_task_prefect_logs_from_deployments(self, org: str, s3):
                         if task_slug is None:
                             continue
 
+                        # set status of the task run
+                        if task_slug == "dbt-test":
+                            if (
+                                task_logs["state_type"] == "FAILED"
+                                or flow_run["state_name"] == "DBT_TEST_FAILED"
+                            ):
+                                status = self.FAILURE_LOGS
+                            elif task_logs["state_type"] == "COMPLETED":
+                                status = self.SUCCESS_LOGS
+                        elif task_slug == "dbt-run":
+                            if task_logs["state_type"] == "COMPLETED":
+                                status = self.SUCCESS_LOGS
+                            elif task_logs["state_type"] == "FAILED":
+                                status = self.FAILURE_LOGS
+
                         # upload logs for the current flow_run_id under the task_slug
                         flow_run_logs = task_logs["logs"]
-                        s3key = f"logs/dbt/{org}/{flow_run_id}/{task_slug}.json"
-                        s3.put_object(
-                            Bucket="dalgo-t4dai",
-                            Key=s3key,
-                            Body=json.dumps(flow_run_logs),
-                        )
-                        print(f"Saved s3://dalgo-t4dai/{s3key}")
+
+                        if status is not None:
+                            summ[task_slug][status] += 1
+                            print(
+                                f"Found {status} logs for task : {task_slug} in flow_run : {flow_run_id}"
+                            )
+                            s3key = f"logs/dbt/{org}/{flow_run_id}/{task_slug}/{status}.json"
```

Comment:
do you want to put the status at the beginning of the key

logs/dbt/status/...

Reply:
i asked Jerome, he was fine with this. If he asks for it i will change.